recipe/meta.yaml

```diff
@@ -50,11 +50,11 @@ outputs:
         # above, we need most of these at build time, so they are mostly
         # host-like
         - boost =1.74
-        - cfitsio =3
+        - cfitsio =4
         - eigen =3.3
         - eups
         - firefly-client
-        - galsim =2.2
+        - galsim =2.3
```

Comment:
@ktlim will need to approve these pin changes.

Reply:
This is meant as a draft with a tentative version number `0.8.0` for testing. 
`galsim` and `cfitsio` can be unpinned and would currently resolve to the versions above. Pinning to the previous versions would result in a conflict for `aarch64`

Reply:
cfitsio is not at 4 yet. we have an ongoing migration: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L393